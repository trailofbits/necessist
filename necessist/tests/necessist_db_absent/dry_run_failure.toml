args = ["--no-sqlite", "--root=examples/dry_run_failure"]
stderr = ""

[bin]
name = "necessist"

[fs]
cwd = "../../.."

[status]
code = 0
