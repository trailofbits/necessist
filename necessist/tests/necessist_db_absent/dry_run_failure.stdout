2 candidates in 2 tests in 3 source files
fixtures/dry_run_failure/tests/a.rs: dry running
fixtures/dry_run_failure/tests/a.rs: Warning: dry run failed: code=101
stdout=```

running 1 test
test dry_run_failed ... FAILED

failures:

---- dry_run_failed stdout ----

thread /'dry_run_failed/' panicked at [..]/a.rs:5:5:
assertion failed: n >= 2
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace


failures:
    dry_run_failed

test result: FAILED. 0 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out; finished in [..]s

```

stderr=```
...
error: test failed, to rerun pass `--test a`
```


Silence this warning with: --allow dry-run-failed
fixtures/dry_run_failure/tests/b.rs: dry running
fixtures/dry_run_failure/tests/b.rs: mutilating
fixtures/dry_run_failure/tests/b.rs:4:5-4:12: `n += 1;` passed
